app/main/views/agreements.py

```diff
@@ -8,10 +8,8 @@
 )
 
 from ...utils import (
-    get_json_from_request,
-    json_has_required_keys,
-    json_has_keys,
-    validate_and_return_updater_request,
+    get_json_from_request, json_has_required_keys, json_has_keys,
+    validate_and_return_updater_request, single_result_response
```

Comment:
Formatting wise, much *much* prefer the former. Far *far* more copy-pastable and easier to see what's present and what's absent from a quick glance.